### ISSUE TITLE
Mais ideias para o jogo, MASTER SOURCE

### ISSUE THREAD

**Issue by luisvmjr** [luisvmjr/JogoAPDA#5]
Sugiram a baixo:

**Comment by Marcos-Oleiro** [comment]
Item 2.5:

Acho que perder itens na morte é demais. Perder xp até vai, mas itens acho punição demais.

**Comment by Marcos-Oleiro** [comment]
Tipo, no wow o cara perde parte da durabilidade dos itens q ta usando e só....e tem q voltar no lugar onde morreu e achar o corpo. Não tô falando pra fazer igual. Mas sei sei lá, perde 1% da xp necessária pra avançar 1 lvl e só.
Exemplo: o cara tá 34..e a xp pra passar do 34 pro 35 é 100....ele perde 1 xp...ou aumenta a porcentagem...ae teria q fazer umas suposições pra ver até onde fica justo.

**Comment by mvaipa** [comment]
perder itens e foda, eu to la dropando itens da hora, ai vou la e morro e pah... perco tudo que tenho!!

**Comment by datafanatic** [comment]
Perder items é algo que precisa ser muito estudado antes de ao menos considerar colocar em prática, é uma punição muito rígida especialmente dependendo da dificuldade pra conseguir equipamentos bons.

Tenho essas propostas:

* Perder uma porcentagem da XP ou dano nos equipamentos como proposto pelo @Marcos-Oleiro em https://github.com/luisvmjr/JogoAPDA/issues/5#issuecomment-167379577;
* Tempo morto antes de reviver
  ** 10 a 15 minutos,
  ** 20 a 30 minutos;
* Perda de possíveis buffs que o jogador possa ter;
* Efeito Maldição 
 ** Jogador renasce imediatamente mas não pode usar poções de vida durante os próximos 30 ou 40 minutos;
* Perda de bravura (proposto no item 6 de https://github.com/luisvmjr/JogoAPDA/issues/1#issuecomment-167196380);

**Comment by datafanatic** [comment]
2.1 - Por qual motivo a ideia das vocações foi colocada de lado?

**Comment by Marcos-Oleiro** [comment]
@datafanatic 
Não acho que tempo morto seja necessário, o cara tá la no animo pra caçar com os amigos ae dá lag e morre e pronto, fudeu, 10min morto, acabou a diversão. Achoi que só punição na xp e até em skills já dá.

**Comment by datafanatic** [comment]
@Marcos-Oleiro São propostas de punições diferentes, só uma ou duas seria aplicada no jogo. Acho que não fui claro no meu comentário.

**Comment by Marcos-Oleiro** [comment]
@datafanatic 
Foi sim cara, eu voto na perda de xp somente.

**Comment by jhonatanwestphal** [comment]
Acho que o tempo sem jogar, acabaria tirando o player do jogo.
Creio que podia voltar pro começo do level que estava, se não quiser perder item. Para rolar mais seriedade na hora de tentar "explorar" algo, assim fica mais interessante.

Acho que podíamos explorar um pouco mais a rivalidade dentro do jogo, algo relacionado a domínio de vilas, mas controlado. Essas vilas poderiam guerriar, ou receber ataques de vilas orcs/trolls/entes/dragoes.

Cada vila pode gerar grana à guild que domina, porém recebe ataque o tempo todo e desafios constantes. As próprias vilas também poderiam gerar quests à população, algo como trazer a cabeça de determinado líder, espada roubada, enfim. Caso a guild perca pros vilarejos, estes ficarão de certa forma "escravizados" (de forma que as vendas paguem taxa, não seja permitido entrar em alguns lugares, quests rendem menos).

Assim como guerriar, poderíamos estudar formas de alianças, receptividade à vila, passagem a lugares diferentes, quests diferentes, inimigos diferentes (inclusive determinada vila de players).
Para ser "residente" de uma vila, poderia haver algum controle, algo com ser residente do local, permanecer um certo tempo na vila, etc..

Só não pode focar só em High Level, que senão fica massante...

O que acham?

**Comment by Marcos-Oleiro** [comment]
Bom, nessa parte de PVP eu não vou dar muito pitaco pq eu não curto muito pvp em mmo.Eu acho que o pvp tem q ser 'controlado', um jogador não pode sair matando geral só pq ele qr 'diversão' ou teve um dia de cú. Acho que o pvp tem q ser feito  em arenas, áreas destinadas para isso ou se as partes aceitarem. 
Eu nunca curti muito esse treco de domínio, pagar taxa ao dominante, mas isso eu falo como jogador que sempre quis só jogar em paz mas eu entendo que tem muita gente que curte essa 'disputa de poder'.
Quanto à invasões de mobs, acho bom sim, uma putaria na vila é legal, traz um sentimento de união (tipo 'vamos proteger nossa vila!').

**Comment by jhonatanwestphal** [comment]
@Marcos-Oleiro também costumo curtir ficar mais de boa, mas se for só upar fica sem graça sabe? É legal aquele "Vou chamar meu amigo" haha. Com relação às vilas, acho que a briga entre vilas é legal, porque é tipo o lance de arena, só que com vilas, é basicamente uma WAR mas com a intenção de unir mais a história..

Quanto à taxa, não é a população que paga, é o próprio jogo para ajudar a bancar as "WARS" com outras vilas, orcs, minos, tals. A população só paga, se os mobs dominarem e essa a intenção, tentar manter o poder com os players...

**Comment by datafanatic** [comment]
@jhonatanwestphal Eu gostei bastante das ideias mas fiquei pensando em uma coisa em relação às vilas. Por exemplo, se um exército de mobs toma controle de uma vila, o que impede os jogadores (leia-se a população daquela vila) e simplesmente sair e ir pra outra vila que não foi dominada por mobs? Eu não tenho interesse em forçar os jogadores a ficarem numa cidade que eles não querem, mas, olha, em pouco tempo teríamos dezenas de cidades fantasmas pelo jogo, não? Eu acho que isso ficaria bem legal se todas as construções da vila fossem feitas pelos jogadores, como foi proposto no #1 pois aí se os jogadores não quisessem retomar a vila, eles sairiam no prejuízo.

**Comment by jhonatanwestphal** [comment]
Sim, gostei da ideia das vilas secundárias serem construídas pelos jogadores.
Poderia ser dificultada a troca de vilarejo, senão vai rolar muito "cavalo de troia" nas guerras de vila.
Mas acho que você mesmo já achou a solução, o gasto vai ser grande pra quem desertar.

**Comment by davidasimoes** [comment]
Queria dar uma sugestão para as armas de tipo no jogo "DDtank", não sei se alguem já viu, mas poderia ter pedras(ou outro ítem) para fortalecer as armas.

**Comment by datafanatic** [comment]
Oi, @davidasimoes, para nós entendermos a mecânica da sua sugestão, é necessário que fale um pouco mais sobre ela, explique como funcionava, o propósito; etc.

**Comment by jhonatanwestphal** [comment]
Creio ser sistema de aprimoramento de armas através de runas, com chance de quebrar e tals. Pode ser incorporado na forja, inclusive na criação dos itens.

**Comment by davidasimoes** [comment]
Olá @jhonatanwestphal , é exatamente isto que vc comentou!
